.rubocop.yml

```diff
@@ -1,10 +1,10 @@
 inherit_from: .rubocop_todo.yml
 
-require:
+plugins:
   - rubocop-performance
   - rubocop-rake
```

Comment:
These two don’t seem to be plugins just yet.

Reply:
RuboCop Performance 1.24 with plugin support has been released:
https://github.com/rubocop/rubocop-performance/releases/tag/v1.24.0

Reply:
👍🏼 Thanks